app/utils/string.js

```diff
@@ -49,5 +49,5 @@ export function getLastUrlFromText(text) {
  * @return {string}
  */
 export function escapeRegex(text) {
-	return text.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&');
```

Comment:
Are you sure about this? How is it related to the update?

Reply:
updated eslint rules, no-useless-escape